artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -2764,6 +2764,35 @@ private JsonObject toJSONObject(ServerConsumer consumer) throws Exception {
       }
    }
 
+   @Override
+   public Object[] getAcceptors() throws Exception {
+      if (AuditLogger.isBaseLoggingEnabled()) {
+         AuditLogger.getAcceptors(this.server);
+      }
+      checkStarted();
+
+      clearIO();
+      try {
+         Collection<TransportConfiguration> acceptorConfigurations = configuration.getAcceptorConfigurations();
+
+         Object[] ret = new Object[acceptorConfigurations.size()];
+
+         int i = 0;
+         for (TransportConfiguration config : acceptorConfigurations) {
```

Comment:
this code is very similar to the code of `getConnectors`, is it possible avoiding this code duplication?